editor/src/core/workers/common/import-worker-utils.ts

```diff
@@ -0,0 +1,237 @@
+import type { Imports, ImportsMergeResolution } from '../../../core/shared/project-file-types'
```

Comment:
This file shouldn't be in the `workers` code since it isn't specifically for the workers. I'd say move it to /core/shared (and rename it to remove the "workers" in the name)